src/api/app/components/bs_request_comment_component.rb

```diff
@@ -19,4 +19,17 @@ def range
     line_index = @comment.diff_ref.match(/diff_[0-9]+_n([0-9]+)/).captures.first
     ((line_index.to_i - 4).clamp(0..)..(line_index.to_i - 1))
   end
+
+  def outdated?
+    return false unless @comment.diff_ref.present?
+    return true if @comment.source_rev.nil? && @comment.target_rev.nil?
```

Comment:
```suggestion
    return false if @comment.source_rev.nil? && @comment.target_rev.nil?
```
Shouldn't we return `false` in this case, instead? I'm thinking about the inline comments created before introducing the `source_rev` and `target_rev` fields. For those recrods, `@comment.diff_ref.present?` is `true` and also `@comment.source_rev.nil? && @comment.target_rev.nil?` is `true`.

Reply:
Yeah, for that we simply don't have enough info to tell if they are outdated, it might be a better idea to treat them like that instead

Reply:
I prefer to treat them as not outdated. We are introducing outdated diffs from now on. I don't find better to show Outdated in all the previous comments...

Reply:
Done